modules/imgaug/include/opencv2/imgaug/functional.hpp

```diff
@@ -0,0 +1,46 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+#ifndef OPENCV_AUG_FUNCTIONAL_HPP
+#define OPENCV_AUG_FUNCTIONAL_HPP
+#include <opencv2/core.hpp>
+#include <vector>
+
+namespace cv {
+    //! @addtogroup imgaug
+    //! @{
+
+    /** @brief Adjust the brightness of the given image.
+     *
+     * @param img Source image. This operation is inplace.
+     * @param brightness_factor brightness factor which controls the brightness of the adjusted image.
+     * Brightness factor should be >= 0. When brightness factor is larger than 1, the output image will be brighter than original.
+     * When brightness factor is less than 1, the output image will be darker than original.
+     */
+    void adjustBrightness(Mat& img, double brightness_factor);
```

Comment:
It makes sense to use `InputArray` and `OutputArray` for all functions in this header. Rationale:
- Generic OpenCV interface.
- UMat support, if `InputArray` is promoted from `Transform` classes directly without getMat() call.